src/views/utils.js

```diff
@@ -65,36 +65,40 @@ export function getImageLayers(id, props) {
     ...layerProps
   } = props;
   const { loader } = layerProps;
+
   // Create at least one layer even without loaderSelection so that the tests pass.
-  if (loader.isPyramid) {
-    return [loaderSelection, newLoaderSelection]
-      .filter((s, i) => i === 0 || s)
-      .map((s, i) => {
-        const suffix = s
-          ? `-${transitionFields.map(f => s[0][f]).join('-')}`
-          : '';
-        const newProps =
-          i !== 0
-            ? {
-                onViewportLoad,
-                refinementStrategy: 'never',
-                excludeBackground: true
-              }
-            : {};
-        return new MultiscaleImageLayer({
-          ...layerProps,
-          ...newProps,
-          loaderSelection: s,
-          id: `${loader.type}${getVivId(id)}${suffix}`,
-          viewportId: id
-        });
-      });
+  if (loader.length === 1) {
+    return [
+      new ImageLayer(layerProps, {
+        id: `${loader.type}${getVivId(id)}`,
```

Comment:
Need to change this `loader.type` since it doesn't currently exist.  Might be worth adding this to the interface, even optionally, though.

Reply:
I want to be super sparing in terms of what gets added to the loader interface (even optionally). The idea is that it shouldn't matter. That said I added this bit of logic:

```javascript
  // Grab name of PixelSource if a class instance (works for Tiff & Zarr).
  const sourceName = loader[0]?.constructor?.name;
```

Reply:
Which will use the instance name if it's a class, otherwise it's undefined. Kind of a "hidden" property which is useful to for layer-debugging, but not something that is exposed via any normal public API.